code/game/objects/machinery/computer/camera_advanced.dm

```diff
@@ -263,27 +269,20 @@
 
 
 /mob/camera/aiEye/remote/relaymove(mob/user, direct)
-	var/initial = initial(sprint)
-	var/max_sprint = 50
-
 	if(istype(origin, /obj/machinery/computer/camera_advanced))
 		var/obj/machinery/computer/camera_advanced/CA = origin
 		CA.tracking_target = null
-
-	if(cooldown && cooldown < world.timeofday) // 3 seconds
-		sprint = initial
-
-	for(var/i = 0; i < max(sprint, initial); i += 20)
-		var/turf/T = get_turf(get_step(src, direct))
-		if(T)
-			setLoc(T)
-
-	cooldown = world.timeofday + 0.5 SECONDS
-	if(acceleration)
-		sprint = min(sprint + 0.5, max_sprint)
-	else
-		sprint = initial
-
+	if(cooldown > world.time)
+		return
+	acceleration = ((cooldown + move_delay * 5) > world.time) ? acceleration : 0
+	cooldown = world.time +	move_delay
+	cooldown -= acceleration ? tiles_moved * (move_delay / 10) : 0
```

Comment:
Why are you doing it in two lines

Reply:
moved it onto 1 line. It was mainly for clarity.